app/service.js

```diff
@@ -42,13 +42,17 @@ function getDocumentMetadata(url, window) {
     title: metadata.title,
     description: metadata.description,
     favicon_url: metadata.icon_url ? makeUrlAbsolute(url, metadata.icon_url) : makeUrlAbsolute(url, '/favicon.ico'),
-    images: [{
+    images: []
+  };
+
+  if (metadata.image_url) {
+    responseData.images = [{
```

Comment:
This is more of a change for https://github.com/mozilla/page-metadata-parser, since right now it only supports finding a single preview image.  It's certainly possible to return multiple images, in theory, but it's outside the design and use case.